subtests/docker_cli/run_cgroup_parent/run_cgroup_parent.py

```diff
@@ -161,9 +161,10 @@ class run_cgroup_parent_invalid_name(run_cgroup_parent_base):
     def initialize(self):
         super(run_cgroup_parent_invalid_name, self).initialize()
         self._setup("/{rand1}")
-        self._expect(stderr="Error response from daemon:"
-                     " Cannot start container {cid}: [8] System error:"
-                     " Invalid slice name /{rand1}")
+        self._expect(stderr="docker: Error response from daemon:"
+                     " cgroup-parent for systemd cgroup should be a"
+                     " valid slice named as \"xxx.slice\".\n"
+                     "See '/usr/bin/docker run --help'.")
```

Comment:
Unf. this happens way more than would make our lives easy.  Consider making the message token into a configuration option value.  Also consider making it a less precise regex,  e.g. ``r'rror response from daemon.+slice name'``.  

Reason:  Configuration changes are **_easy_** to do in CI to suit the situation at hand.  Anything dealing with code/commit/version changes are much more difficult/problematic in CI (possibly affecting other tests as well).  Also, slightly Less precise because it reduces the maintenance burden to a degree disproportional to the risk of false-positives (i.e. not matching when it should have).